core/server/src/main/java/alluxio/cli/ValidateConf.java

```diff
@@ -36,9 +36,10 @@
   /**
    * Validates the configurations.
    *
+   * @param configuration Configuration to validate
    * @return true if the validation succeeds, false otherwise
    */
-  public static boolean validate() {
+  public static boolean validate(Configuration configuration) {
```

Comment:
not really. this method is also invoked by the master and worker. On the main, it is doing client-side check.